shared/src/packages/drivers/libs/validation-schemas/driver-get-page-parameters.validation-schema.ts

```diff
@@ -0,0 +1,19 @@
+import joi from 'joi';
+
+import {
+  type BusinessGetDriversPageRequestQuery,
+  DriverValidationMessage,
+} from '../types/types.js';
+
+const driverGetPageQuery = joi.object<BusinessGetDriversPageRequestQuery, true>(
```

Comment:
this schema can be used not only for drivers table, it should be validation-schemas/common-schemas/get-page-parameters.validation-schema